cookbooks/bcpc-hadoop/recipes/maven_config.rb

```diff
@@ -2,14 +2,15 @@
 
 node.override['maven']['install_java'] = false
 
-internet_download_url = node['maven']['3']['url']
-maven_file = Pathname.new(node['maven']['3']['url']).basename
+internet_download_url = node['maven']['url']
+maven_file = Pathname.new(node['maven']['url']).basename
 
-node.override['maven']['3']['url'] = "#{get_binary_server_url}/#{maven_file}"
+node.override['maven']['url'] = "#{get_binary_server_url}/#{maven_file}"
 
 # download Maven only if not already stashed in the bins directory
 remote_file "/home/vagrant/chef-bcpc/bins/#{maven_file}" do
   source internet_download_url
-  action :create_if_missing
-  mode '0555'
+  action :create
```

Comment:
On a disconnected cluster would this cause grumpiness? If so perhaps the URL variable should not be `internet_download_url` but maven tarball URL or some such (implying one should set it to a local URL)?